deployment/frontend/src/components/datasets/DatasetHeader.tsx

```diff
@@ -103,16 +106,46 @@ export function DatasetHeader() {
           </div>
         </div>
       </div>
-      <div className="flex gap-x-2 pb-10">
-        <span className="flex h-7 w-fit items-center justify-center rounded-sm bg-wri-light-green px-3 text-center text-xs font-normal text-black">
-          <span className="my-auto">TIFF</span>
-        </span>
-        <span className="flex h-7 w-fit items-center justify-center rounded-sm bg-wri-light-yellow px-3 text-center text-xs font-normal text-black">
-          <span className="my-auto">CSV</span>
-        </span>
-        <span className="flex h-7 w-fit items-center justify-center rounded-sm bg-wri-light-blue px-3 text-center text-xs font-normal text-black">
-          <span className="my-auto">GeoJSON</span>
-        </span>
+      <div className="flex lg:flex-row flex-col items-center gap-x-3 rounded-sm bg-cyan-700 bg-opacity-10 p-3">
+        <ExclamationTriangleIcon className="h-8 w-8 sm:h-12 sm:w-12 text-yellow-600" />
+        <p>
+          <span className="font-acumin text-sm font-semibold leading-none text-black">
+            Caution:{" "}
+          </span>
+          <span className="text-sm font-light text-black">
+            Data set is not for use in litigation. While efforts have been made
+            to ensure that these data are accurate and reliable within the state
+            of the art, WRI, cannot assume liability.{" "}
+          </span>
+          <a
+            href="#"
+            className="font-acumin text-sm font-normal leading-none text-wri-green underline"
+          >
+            Read more
+          </a>
+        </p>
+      </div>
+      <div className="mt-4 flex justify-start gap-x-3">
+        <div className="flex items-center rounded-[3px] border border-green-500 bg-green-500">
+          <div className="px-2 font-acumin text-xs font-medium text-white">
+            RDI approved
+          </div>
+        </div>
+        <div className="flex gap-x-2 border-l border-zinc-300 pl-3">
+          <span className="flex h-7 w-fit items-center justify-center rounded-sm bg-wri-light-green px-3 text-center text-xs font-normal text-black">
+            <span className="my-auto">TIFF</span>
+          </span>
+          <span className="flex h-7 w-fit items-center justify-center rounded-sm bg-wri-light-yellow px-3 text-center text-xs font-normal text-black">
+            <span className="my-auto">CSV</span>
+          </span>
+          <span className="flex h-7 w-fit items-center justify-center rounded-sm bg-wri-light-blue px-3 text-center text-xs font-normal text-black">
+            <span className="my-auto">GeoJSON</span>
+          </span>
+        </div>
+      </div>
+      <div className="flex items-center gap-x-1 pt-4">
+        <LinkIcon className="h-4 w-4 text-wri-green" />
+        <div className="text-green-700 text-sm font-semibold font-['Acumin Pro SemiCondensed']">Technical Notes</div>
```

Comment:
Looking at the mook ups I think this should be only on the about section

Reply:
they have those in two different places, i assumed it was decided yet where they are going to live